src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -2573,7 +2573,7 @@ void EmitCapture(RegexNode node, RegexNode? subsequent = null)
                     }
 
                     writer.WriteLine();
-                    writer.WriteLine($"base.TransferCapture({capnum}, {uncapnum}, {startingPos}, pos);");
+                    writer.WriteLine($"base.TransferCapture({capnum.ToString(CultureInfo.InvariantCulture)}, {uncapnum}, {startingPos}, pos);");
```

Comment:
```suggestion
                    writer.WriteLine(string.Create(CultureInfo.InvariantCulture, $"base.TransferCapture({capnum}, {uncapnum}, {startingPos}, pos);"));
```

Reply:
The generator needs to work on netstandard2.0.